.github/workflows/postgres-ptr.yml

```diff
@@ -0,0 +1,59 @@
+name: Restore Database from point in time
```

Comment:
This looks like the restore workflow but it's very different. Can the name be clearer, like "Restore database from point in time to new database server"

Reply:
changed name